doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -102,6 +102,9 @@ As with the global method above, multiple configuration files may be listed,
 each with its own ``@pparam=<file>`` and their contents will be evaluated in
 the order the files were specified.
 
+Each ruleset that is configured per-mapping must have a special
+`Hook Conditions`_ defined.
```

Comment:
Not strictly true, as rules without a Hook specified use the REMAP_PSEUDO_HOOK.

How about changing the "must" to "should", and adding, " Without a defined hook, these rulesets will use the REMAP_PSEUDO_HOOK."